backend/src/main/scala/cromwell/backend/Backend.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.backend
+
+import cromwell.backend.model.{WorkflowDescriptor, JobDescriptor}
+
+/**
+  * Defines basic structure and functionality to work with a backend.
+  */
+trait Backend {
```

Comment:
2nd: I will remove backend trait to make it simple. Later if we implement PA's modify this. I don't see we will support PA's / EA's in a short term.